dbms/src/AggregateFunctions/AggregateFunctionNull.cpp

```diff
@@ -51,10 +52,15 @@ class AggregateFunctionCombinatorNull final : public IAggregateFunctionCombinato
         if (!has_nullable_types)
             throw Exception("Aggregate function combinator 'Null' requires at least one argument to be Nullable", ErrorCodes::ILLEGAL_TYPE_OF_ARGUMENT);
 
-        /// Special case for 'count' function. It could be called with Nullable arguments
-        /// - that means - count number of calls, when all arguments are not NULL.
-        if (nested_function && nested_function->getName() == "count")
-            return std::make_shared<AggregateFunctionCountNotNullUnary>(arguments[0], params);
+        if (nested_function)
+        {
+            /// Special case for 'count' function. It could be called with Nullable arguments
+            /// - that means - count number of calls, when all arguments are not NULL.
+            if (nested_function->getName() == "count")
+                return std::make_shared<AggregateFunctionCountNotNullUnary>(arguments[0], params);
+            else if (nested_function->getName() == "windowFunnel")
```

Comment:
It looks like unnesessary dependency... it makes code looking weird.

Reply:
@alexey-milovidov 
At the above scenario, I don't know find the events and actions flow of `event='e1' --> action='a1'` without AggregateFunctionWindowFunnelNoNullifyVariadic.
If you give me a little hint, I will try to fix it.
Thank you.